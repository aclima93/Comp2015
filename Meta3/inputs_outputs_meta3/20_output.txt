Line 2, col 24: Type identifier expected
