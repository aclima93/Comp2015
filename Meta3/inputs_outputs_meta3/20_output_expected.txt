===== Outer Symbol Table =====
boolean	_type_	constant	_boolean_
integer	_type_	constant	_integer_
real	_type_	constant	_real_
false	_boolean_	constant	_false_
true	_boolean_	constant	_true_
paramcount	_function_
program	_program_

===== Function Symbol Table =====
paramcount	_integer_	return

===== Program Symbol Table =====
gcd	_function_
decla	_function_
gcd123	_function_
declaratio1	_function_

===== Function Symbol Table =====
gcd	_integer_	return
a	_brolly_	varparam
b	_brolly_	varparam
l	_boolean_	param
k213123saa	_boolean_	param
x	_integer_
y	_integer_

===== Function Symbol Table =====
decla	_integer_	return
int	_integer_

===== Function Symbol Table =====
gcd123	_integer_	return
z	_integer_	param
e	_integer_	param
boolean	_integer_
integer	_integer_
integ	_boolean_
paramcount	_boolean_

===== Function Symbol Table =====
declaratio1	_integer_	return
qwerty	_integer_